rules/windows/builtin/win_remote_schtask.yml

```diff
@@ -0,0 +1,36 @@
+title: Remote Schtasks Creation
+id: cf349c4b-99af-40fa-a051-823aa2307a84
+status: experimental
+description: Detects remote execution via scheduled task creation or update on the destination host
+author: Jai Minton, oscd.community
+date: 2020/10/05
+references:
+    - https://drive.google.com/file/d/1lKya3_mLnR3UQuCoiYruO3qgu052_iS_/view
+tags:
+    - attack.lateral_movement
+    - attack.persistence
+    - attack.execution
+    - attack.t1053.005
+logsource:
+    product: windows
+    service: security
+    definition: 'The Advanced Audit Policy setting Object Access > Audit Other Object Access Events has to be configured to allow this detection (not in the baseline recommendations by Microsoft).'
+detection:
+    selection1:
+        EventID: 4624
+        Logon_Type: 3
```

Comment:
Hello @JPMinty !

The `selection1` even with filter1 applied will trigger a huge amount of events, because 4624 is "logon" and Logon_Type 3 is a network logon. Such events are absolutely normal in AD environments and will not help to detect the technique. 

Most probably, the original Splunk query works fine because of the operations with the pipes (transaction etc):

<img width="1146" alt="Screenshot 2020-10-08 at 21 57 10" src="https://user-images.githubusercontent.com/13090109/95507127-410a7d80-09b1-11eb-9884-7420a162da43.png">

I've tried to analyze the Splunk query, but honestly, it's out of my area of knowledge.
Let's try to ask the author. 

summon @Heirhabarov

Reply:
HI guys! This query looks for events 4698 or 4702 that follow successful network logon events, where the SubjectLogonId from event 4698/4702 is the same as the TargetLogonId from event 4624.
![image](https://user-images.githubusercontent.com/33382447/95517732-3dcfbb80-09ca-11eb-8938-32e9ed04d218.png)

Reply:
Yeah agreed @yugoslavskiy . It's broad as it is and will likely have a lot of noise.

Thanks for the update Heirhabarov! I'm trying to think on how this would look in a Sigma rule and how it can be done to recreate transaction. I'm thinking we need to get the Logon IDs from 4698 and use these in a sub-search to find the related 4624 Logon event from these entries at which point a type of 3 would match what we're after.
Exactly how this looks as a Sigma rule I'm not too sure. More than happy for suggestions.

Reply:
Contributing to the `rules-unsupported` is quite an important step that helps to support the development of the new sigmac converter capabilities. So please do this change for this PR as well as #1034, it will be very beneficial for the community. 

Thank you, @JPMinty!

Reply:
Thankyou @e6e6e and @yugoslavskiy for the update!
I've gone ahead and moved this to rules-unsupported and added a description of the proposed logic based on how transaction works and done the same on #1034 
Greatly appreciated.

Reply:
Thank you for your contribution, @JPMinty!